src/util/real_algebraic_number_poly_imp.h

```diff
@@ -0,0 +1,160 @@
+/*********************                                                        */
+/*! \file real_algebraic_number_poly_imp.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Gereon Kremer
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Algebraic number constants; wraps a libpoly algebraic number.
+ **
+ ** Algebraic number constants; wraps a libpoly algebraic number.
+ **/
+
+#include "cvc4_public.h"
```

Comment:
Can we make this header private, too? I think that it would be good to not introduce more public headers. In the (near) future, we are planning to only have stuff in `src/api` be public. If we temporarily need this to be public, it is ok, though.